suite-native/transactions/src/components/TransactionDetail/TransactionDetailSummary.tsx

```diff
@@ -97,24 +117,36 @@ export const TransactionDetailSummary = ({
     return (
         <Card>
             <VStack style={applyStyle(cardContentStyle)}>
-                <TransactionDetailSummaryRow value={origin} title="From" />
-                <VerticalSeparator />
-                <Box flexDirection="row" alignItems="center">
-                    <Box style={applyStyle(confirmationIconStyle, { isTransactionPending })}>
-                        <Icon
-                            name={isTransactionPending ? 'clockClockwise' : 'check'}
-                            color="gray0"
-                        />
-                    </Box>
+                <SummaryRow leftComponent={<TransactionDetailSummaryStepper />}>
+                    <RowWithTitle title="From" value={origin} />
+                </SummaryRow>
+                <SummaryRow
+                    leftComponent={
+                        <Box alignItems="center" style={{ width: 40 }}>
```

Comment:
Width should not be necessary here

Reply:
https://github.com/trezor/trezor-suite/pull/7272/commits/39ce90eab3ceee16f586a865709b88eb24350166